walkthroughs/week-3-server/examples/text-processor/src/main/java/com/google/sps/servlets/TextProcessorServlet.java

```diff
@@ -48,6 +48,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Respond with the result.
     response.setContentType("text/html;");
     response.getWriter().println(Arrays.toString(words));
+    response.sendRedirect("https://google.com");
```

Comment:
I'm not sure why we need this redirect and I'm not sure what is using the `/text` endpoint.

Reply:
It is just to test out sendRedirect() function for the tutorial. It's asked to add the link with no additional changes.